interface/super/edit_layout.php

```diff
@@ -1191,8 +1191,9 @@ function cidChanged(lino, seq) {
 
 // This invokes the popup to edit layout properties or add a new layout.
 function edit_layout_props(groupid) {
+ var title = "<?php echo xlt('Layout Properties');?>";
  dlgopen('edit_layout_props.php?layout_id=<?php echo attr($layout_id); ?>&group_id=' + groupid,
-  '_blank', 700, 550);
+  '_blank', 'modal-sm', 550, '', title);
```

Comment:
likely stupid question.
If this is used in frames mode, will the popup know the size for modal-sm?

Reply:
Not stupid. I changed those back to absolute values when decided not to use on frames but forgot to include in my last update sooo, it's covered.
Concerning spinner, it really should be in body and not header but in reality i'm just going to remove as dialogs really are not loading much content to worry with and I think instead of zero height as starting point for dialog, i'll set initial height to function call height and size up or down from there. One way or the other, I want dialogs to size to view port :)